templates/mail/waitlist.html

```diff
@@ -0,0 +1,45 @@
+{% extends 'base.html' %} {% block body %}
+<h1>Join the waitlist</h1>
+<div id="mcWaitlistForm">
+  <!-- Begin Mailchimp Waitlist Form -->
+  <form
+    action="#"
+    class="waitlist-form"
+    method="post"
+    id="mc-embedded-waitlist-form"
+    name="mc-embedded-waitlist-form"
+  >
```

Comment:
I think this is suppose to have a valid action? If so can you add it to an env val, pipe it through to settings and add it as a template context? 

I do something similar with connection info at https://github.com/thunderbird/thunderbird-accounts/pull/14/files#diff-f9a2b1bfd2b55e873305640e2c285d3e3ff850edece24ab036e94001a2c61312R39

Reply:
I just copied the `<form>` for the newsletter signup from thunderbird.net  -- we haven't created anything on the MC side, so I removed the `action`. Also, I'm unsure whether the actual form code from MC will be structured differently.

That said, I can put a value in an env val (using a dummy for now) and use that via template context

Reply:
Oh the mailchimp account should be in 1password, feel free and connect with @inattee to create a list.